purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot

```diff
@@ -0,0 +1,143 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* purchase_order_approval_block
```

Comment:
maybe you have to regenerate it again ;)

Reply:
Yes done!